docs/content/howto/helm_chart_redhat_certification.md

```diff
@@ -2,6 +2,8 @@
 
 This document covers how to prepare and Red Hat certify your Helm operator for the goal of being added to [Red Hat Marketplace](https://marketplace.redhat.com/en-us).
 
+> Note: It is recommended if you want to certify a Helm operator that you own and maintain all charts and images that will be managed by the operator. The reason being is, if you have to modify the images and/or the charts to be compliant.
```

Comment:
@hickeyma may be it's just me, and if so please disregard the comment but I found this Note hard to read. May a re-write something like this `It is recommended that you own and maintain all charts and images that will be managed by the operator. Third-party owned content can be difficult to modify and being complaint.`  (I am removing "if you want to certify a Helm Operator" because the Note is in the context of the sentence about certification.)

Reply:
Thanks for the recommendation. I think that it reads better. Updated.